Zend/tests/clone_initializer/clone_initializer_success2.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Test that dynamic properties can be initialized during cloning
```

Comment:
Given that 8.2 deprecates dynamic properties absent an attribute, do we even want to support this?  I could maybe see it if there is a __set defined, but absent that, I'm not sure this is a good feature to include.

Reply:
Yes, this was my plan initially, but then I realized that it would be both cumbersome and inefficient to alter from the current standard behavior due to technical reasons.